core/plugins/org.fusesource.ide.camel.validation/src/org/fusesource/ide/camel/validation/model/NumberValidator.java

```diff
@@ -30,22 +30,35 @@
 	public NumberValidator(Parameter parameter) {
 		this.parameter = parameter;
 	}
+	
+	public NumberValidator() {
+	}
 
-	/* (non-Javadoc)
-	 * @see org.eclipse.core.databinding.validation.IValidator#validate(java.lang.Object)
-	 */
 	@Override
 	public IStatus validate(Object value) {
-		// only check non-empty fields
-		if (CamelComponentUtils.isNumberProperty(parameter)) {
-			if (value != null && value.toString().trim().length() > 0) {
-				try {
-					PropertiesUtils.validateDuration(value.toString());
-				} catch (IllegalArgumentException ex) {
+		if (isNonEmpty(value)
```

Comment:
how about using !Strings.isBlank(value) instead?

Reply:
String.isBlank is not checking for value containing only spaces